<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>somma</class>
 <widget class="QWidget" name="somma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Somma</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lb_Titolo">
        <property name="text">
         <string>Somma 2 interi</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="te_Intero1">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>71</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="te_Intero2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>71</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lb_Somma">
        <property name="text">
         <string>Somma:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_Somma">
        <property name="text">
         <string>SUM</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
